Gopkg.toml

```diff
@@ -73,6 +73,10 @@
   name = "github.com/RichardKnop/redsync"
   branch = "master"
 
+[[constraint]]
+  name = "github.com/baocaixiong/cmq-golang-sdk"
+  branch = "master"
```

Comment:
Tracking master branch is bad because of potential unexpected breaking changes. I see that you use your own package, so I recommend to start versioning it properly and lock dependency on the minor version.